registry.json

```diff
@@ -850,7 +850,19 @@
     "github": ""
   },
  {
- "ticker": "CHAIN",                                                       
+    "ticker": "DIYA",
+    "poolId": "bf8892369c5922d9d5e404ab733c9dbf75b6fb842fa1c09f46c60616",
+    "social": {
+      "facebook": "",
+      "twitter": "@bliss_cs",
+      "website": "https://diya.blissweb.co",
```

Comment:
Your website seems to be offline.

Reply:
It might be some time before I push things on my website.

I can make it to returns the`200` HTTP code and some text for now. But the full version should be coming later.
I Will do that and then notify you on the telegram.